### ISSUE TITLE
toggling a feature to disable deletes the parameters?

### ISSUE THREAD

**Issue by arashbi** [togglz/togglz#273]
When I feature disabled in the admins main page, all its state is removed from database. This causes problems especially if you want to add Actication strategy and then enable the feature.

**Comment by chkal** [comment]
Yes, that's how it is currently implemented. Unfortunately this isn't easy to change, because most state repositories also persist parameters only if the strategy is active.

**Comment by annaira** [comment]
I would like to push this issue as we have had a lot of problems with parameters deleted by accident.
